nixos/modules/services/audio/mpd.nix

```diff
@@ -144,6 +146,7 @@ in {
   ###### implementation
 
   config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.mpd ];
```

Comment:
Why is this required?

Reply:
Not _strictly_ required but it installs the man pages and the documentation which I thought might be useful.